frontend/src/pages/CalendarPage/index.js

```diff
@@ -101,6 +104,25 @@ class CalendarPage extends React.Component {
     });
   }
 
+  handleUpdateStatus = async (
+    values, newStatus
+  ) => {
+    const eventRequest = {
+      ...values,
+      hosts: values.hosts.map(host => {
+        return {
+          id: host.id,
+        };
+      }),
+    };
+    await updateEvent(values.id, eventRequest);
+    const { eventStatusChanged, selectedEvent } = this.state;
+    this.setState({
+      eventStatusChanged: !eventStatusChanged,
+      selectedEvent: {...selectedEvent, status: newStatus}
```

Comment:
Should use the result of updateEvent here. You can create a const var storing the result of updateEvent...

Reply:
do you mean I should use it for selectedEvent or both?